hudi-cli/src/main/java/org/apache/hudi/cli/commands/ArchivedCommitsCommand.java

```diff
@@ -241,6 +241,7 @@ private Comparable[] readCommit(GenericRecord record, boolean skipMetadata) {
       case HoodieTimeline.COMPACTION_ACTION:
         return commitDetail(record, "hoodieCompactionMetadata", skipMetadata);
       case HoodieTimeline.REPLACE_COMMIT_ACTION:
+      case HoodieTimeline.CLUSTER_ACTION:
```

Comment:
I would go with "CLUSTERING_ACTION", since we named "COMPACTION_ACTION" and not "COMPACT_ACTION". 

also, verb makes sense for timeline apis
getPendingCompactionTimeline vs getPendingCompactTimeline
getPendingClusteringTimeline vs getPendingClusterTimeline

Reply:
Addressed